tests/helpers.py

```diff
@@ -949,6 +949,86 @@ def measure_pvc_creation_time(interface, pvc_name):
     return total.total_seconds()
 
 
+def measure_pvc_creation_time_bulk(interface, pvc_obj_list):
+    """
+    Measure PVC creation time of bulk PVC based on logs.
+
+    Args:
+        interface (str): The interface backed the PVC
+        pvc_obj_list (list): List of PVC Names for measuring creation time
+
+    Returns:
+        pvc_dict (dict): Dictionary of pvc_name with creation time.
+
+    """
+    # Get the correct provisioner pod based on the interface
+    pod_name = pod.get_csi_provisioner_pod(interface)
+    # get the logs from the csi-provisioner containers
+    logs = pod.get_pod_logs(pod_name[0], 'csi-provisioner')
+    logs += pod.get_pod_logs(pod_name[1], 'csi-provisioner')
+
+    pvc_dict = dict()
+    format = '%H:%M:%S.%f'
+    for pvc_obj in pvc_obj_list:
+        logs = logs.split("\n")
```

Comment:
can you move this outside for loop? We don't want to split the static log file every time.

Reply:
ack